tests/grid/merge_triangulations_05.output

```diff
@@ -0,0 +1 @@
+
```

Comment:
you probably forgot to add output.

Reply:
Never forget to flush `deallog`! Fixed now.